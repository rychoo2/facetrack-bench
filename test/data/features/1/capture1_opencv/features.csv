filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_eye_distance_x,rel_eye_distance_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191123164037.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5036526150173611,0.3521821952160494,20191123164037000,0.4765625,0.8527777777777777,0.0515625,0.09166666666666666,0.4408162434895833,0.6422831217447916,0.3750268184777462,0.05603859641335227,0.48760330578512395,0.38461538461538464,0.4628099173553719,0.4166666666666667
images/20191123164036.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5036526150173611,0.35224006558641974,20191123164036000,0.415625,0.34444444444444444,0.04921875,0.0875,0.5106714642237103,0.655364990234375,0.47469075520833515,0.005704365079365079,0.4915254237288136,0.41025641025641024,0.4793388429752066,0.4166666666666667
images/20191123164054.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.7553154839409723,0.4531418788580247,20191123164054000,0.4015625,0.44027777777777777,0.0296875,0.05277777777777778,0.5033601459703947,0.6790064761513158,0.5298799213610167,0.020613820929276317,0.46153846153846156,0.6,,
images/20191123164040.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.49724392361111114,0.07240306712962963,20191123164040000,0.43984375,0.6125,0.03125,0.05555555555555555,0.525634765625,0.6807945251464844,0.48909759521484375,0.02385711669921875,0.5042735042735043,0.42424242424242425,0.4661016949152542,0.3939393939393939
images/20191123164108.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.41856689453125,0.5204041280864198,20191123164108000,0.4171875,0.33611111111111114,0.05,0.08888888888888889,0.5194540023803711,0.6597633361816406,0.5046272277832049,-0.0027627944946289062,,,0.4745762711864407,0.43333333333333335
images/20191123164120.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.04120279947916667,0.16381655092592592,20191123164120000,,,,,,,,,,,0.5,0.36363636363636365
images/20191123164121.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.0030748155381944446,0.07585841049382716,20191123164121000,0.41875,0.3333333333333333,0.0484375,0.08611111111111111,0.5434609690020161,0.6846992738785282,0.5630877094884073,-0.037073443012853737,,,,
images/20191123164109.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.3130696614583333,0.6255497685185185,20191123164109000,,,,,,,,,,,0.48717948717948717,0.43333333333333335
images/20191123164041.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4946072048611111,0.008538290895061728,20191123164041000,0.42109375,0.3458333333333333,0.04609375,0.08194444444444444,0.523609226032839,0.6229154942399364,0.4779735500529661,-0.01300824698755393,0.5,0.42424242424242425,0.4700854700854701,0.42424242424242425
images/20191123164055.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.8617404513888889,0.4562958140432099,20191123164055000,,,,,,,,,0.46846846846846846,0.45454545454545453,0.41025641025641024,0.5
images/20191123164057.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9966267903645833,0.4535228587962963,20191123164057000,0.2984375,0.4166666666666667,0.0359375,0.06388888888888888,0.5769971764605978,0.6770026165506114,0.5449046259341033,-0.0038067361582880435,0.4473684210526316,0.5,0.39316239316239315,0.48484848484848486
images/20191123164122.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,,,,0.625,0.20277777777777778,0.03515625,0.0625,0.5008884006076388,0.5986796061197917,0.43024631076388886,0.009174262152777777,,,0.5042735042735043,0.3888888888888889
images/20191123164056.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9674831814236111,0.4529393325617284,20191123164056000,0.47734375,0.8472222222222222,0.0578125,0.10277777777777777,0.4622547046558277,0.6112093538851351,0.37017574825802363,0.053669697529560814,0.4649122807017544,0.4666666666666667,0.39473684210526316,0.5
images/20191123164046.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5010498046875,0.9030092592592592,20191123164046000,0.4921875,0.9236111111111112,0.04140625,0.07361111111111111,0.5380352668042453,0.6587973540683962,0.5811341483637972,-0.058546534124410375,0.5045045045045045,0.38461538461538464,0.4473684210526316,0.4074074074074074
images/20191123164053.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5678941514756944,0.4632233796296296,20191123164053000,0.4234375,0.3472222222222222,0.04453125,0.07916666666666666,0.5213055527001097,0.6357978686951754,0.5331474438048266,0.07051916289748006,,,0.4358974358974359,0.5
images/20191123164053.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.6799370659722223,0.45665991512345677,20191123164053000,0.4234375,0.3472222222222222,0.04453125,0.07916666666666666,0.5213055527001097,0.6357978686951754,0.5331474438048266,0.07051916289748006,,,0.4358974358974359,0.5
images/20191123164045.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5021687825520833,0.7499300733024692,20191123164045000,0.41796875,0.3402777777777778,0.04921875,0.0875,0.5127689422123016,0.6711038256448413,0.5098789760044625,-0.020188588944691558,0.5087719298245614,0.4074074074074074,0.46153846153846156,0.4074074074074074
images/20191123164119.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.12300075954861112,0.26436631944444444,20191123164119000,0.490625,0.9138888888888889,0.0421875,0.075,4.369182445384838,-0.5922772442853009,0.46995487919560186,0.05200534396701389,,,0.5087719298245614,0.3888888888888889
images/20191123164118.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,,,,,,,,,,,,,,0.48717948717948717,0.4358974358974359
images/20191123164044.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4956515842013889,0.4751880787037037,20191123164044000,0.36484375,0.5055555555555555,0.03984375,0.07083333333333333,0.5220348881740197,0.6549533021216299,0.4834277583103565,-0.021121754365808824,,,0.4661016949152542,0.4444444444444444
images/20191123164044.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4989040798611111,0.6463903356481482,20191123164044000,0.36484375,0.5055555555555555,0.03984375,0.07083333333333333,0.5220348881740197,0.6549533021216299,0.4834277583103565,-0.021121754365808824,,,0.4661016949152542,0.4444444444444444
images/20191123164049.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5023749457465277,0.975096450617284,20191123164049000,0.5640625,0.12638888888888888,0.0671875,0.11944444444444445,0.50653076171875,0.597276909406795,0.44981277820675875,0.00870567144349564,,,0.4473684210526316,0.4074074074074074
images/20191123164101.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9917209201388889,0.08969425154320988,20191123164101000,,,,,,,,,,,0.4017094017094017,0.5
images/20191123164115.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.15923258463541667,0.5026982060185186,20191123164115000,0.42109375,0.3388888888888889,0.0453125,0.08055555555555556,0.530181884765625,0.6555459910425646,0.5073810445851293,0.0011854500606152043,,,0.48760330578512395,0.3939393939393939
images/20191123164115.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.24997016059027777,0.49330391589506173,20191123164115000,0.42109375,0.3388888888888889,0.0453125,0.08055555555555556,0.530181884765625,0.6555459910425646,0.5073810445851293,0.0011854500606152043,,,0.48760330578512395,0.3939393939393939
images/20191123164114.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.012440321180555556,0.4955415702160494,20191123164114000,0.4234375,0.34305555555555556,0.04453125,0.07916666666666666,-7.326997857344778,0.10039319490131579,0.42349805329975354,0.05939925344366776,,,0.5087719298245614,0.4074074074074074
images/20191123164100.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9932318793402778,0.19716435185185185,20191123164100000,0.5765625,0.12777777777777777,0.0640625,0.11388888888888889,0.5034395543540396,0.5934446381359566,0.43554464200647863,0.010673988156202362,0.45614035087719296,0.45454545454545453,0.39316239316239315,0.5151515151515151
images/20191123164128.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.19443766276041666,0.2203342013888889,20191123164128000,0.62578125,0.1625,0.03828125,0.06805555555555555,0.49497269610969385,0.6029339225924745,0.4679104551977041,-0.009804239078443877,,,0.49572649572649574,0.4722222222222222
images/20191123164116.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.3717041015625,0.4934968171296296,20191123164116000,,,,,,,,,,,0.4661016949152542,0.4
images/20191123164102.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.8995917426215277,0.1859206211419753,20191123164102000,,,,,,,,,0.46153846153846156,0.5277777777777778,0.4152542372881356,0.5
images/20191123164103.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.8199896918402778,0.267163387345679,20191123164103000,0.07265625,0.2375,0.0984375,0.175,0.5104940505254836,0.6873369973803328,0.5729626246861048,0.0008112589518229166,0.4700854700854701,0.48484848484848486,0.4322033898305085,0.5151515151515151
images/20191123164103.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.7503879123263889,0.3295187114197531,20191123164103000,0.07265625,0.2375,0.0984375,0.175,0.5104940505254836,0.6873369973803328,0.5729626246861048,0.0008112589518229166,0.4700854700854701,0.48484848484848486,0.4322033898305085,0.5151515151515151
images/20191123164117.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.45227186414930554,0.49248890817901236,20191123164117000,,,,,,,,,,,0.4576271186440678,0.3939393939393939
images/20191123164106.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4854342990451389,0.4798948688271605,20191123164106000,,,,,,,,,0.5045045045045045,0.4,0.4661016949152542,0.45454545454545453
images/20191123164112.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.010828993055555555,0.9201244212962963,20191123164112000,0.4578125,0.0,0.190625,0.3388888888888889,0.5435425805263832,0.6400111464203381,0.6104541215740267,-0.025403632492315573,,,0.5042735042735043,0.4
images/20191123164110.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.19084879557291667,0.7333309220679012,20191123164110000,0.465625,0.0,0.175,0.3111111111111111,0.5151361737932477,0.6696515764508929,0.6296048845563617,-0.03710055351257324,,,0.4915254237288136,0.42424242424242425
images/20191123164111.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.06061604817708333,0.8413218557098765,20191123164111000,0.459375,0.0,0.18671875,0.33194444444444443,0.5427470825706067,0.6550560476390886,0.6162543516278766,-0.029890914342393438,,,0.49572649572649574,0.42424242424242425
images/20191123164105.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,,,,0.19375,0.7277777777777777,0.03359375,0.059722222222222225,0.5282947628997093,0.6516482331031976,0.4503989995912077,-0.03686097610828753,0.49122807017543857,0.43333333333333335,0.4576271186440678,0.45454545454545453
images/20191123164059.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9953030056423611,0.314209587191358,20191123164059000,0.00234375,0.6083333333333333,0.03984375,0.07083333333333333,0.4459618961109835,0.6409319709329044,0.5097966661640242,0.006632486979166667,0.45045045045045046,0.5,0.38461538461538464,0.5333333333333333
images/20191123164038.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5004991319444444,0.1840253665123457,20191123164038000,0.42109375,0.3472222222222222,0.04453125,0.07916666666666666,0.5426143177768641,0.636217619243421,0.4871740508497807,-0.0426855254591547,0.5,0.43333333333333335,0.4745762711864407,0.43333333333333335
