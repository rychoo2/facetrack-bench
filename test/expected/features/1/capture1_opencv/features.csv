filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191123164037.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5036526150173611,0.3521821952160494,20191123164037000,0.4765625,0.8527777777777777,0.0515625,0.09166666666666666,0.4393939393939394,0.6363636363636364,1.0303030303030303,-8.5,-0.7272727272727273,-8.393939393939394,1.9242424242424243,-8.272727272727273,0.36363636363636365,0.06060606060606061,0.12121212121212122,-8.166666666666666,0.48760330578512395,0.38461538461538464,0.4628099173553719,0.4166666666666667
images/20191123164036.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5036526150173611,0.35224006558641974,20191123164036000,0.415625,0.34444444444444444,0.04921875,0.0875,0.5079365079365079,0.6507936507936508,2.3333333333333335,-3.0952380952380953,0.4603174603174603,-2.9682539682539684,3.253968253968254,-2.8412698412698414,0.47619047619047616,0.015873015873015872,1.380952380952381,-2.7301587301587302,0.4915254237288136,0.41025641025641024,0.4793388429752066,0.4166666666666667
images/20191123164054.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.7553154839409723,0.4531418788580247,20191123164054000,0.4015625,0.44027777777777777,0.0296875,0.05277777777777778,0.5,0.6578947368421053,4.657894736842105,-7.052631578947368,1.763157894736842,-6.894736842105263,6.078947368421052,-6.578947368421052,0.5526315789473685,0.02631578947368421,3.0789473684210527,-6.2894736842105265,0.46153846153846156,0.6,0.423728813559322,0.8518518518518519
images/20191123164040.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.49724392361111114,0.07240306712962963,20191123164040000,0.43984375,0.6125,0.03125,0.05555555555555555,0.525,0.675,3.025,-9.775,0.175,-9.6,4.5,-9.4,0.5,0.025,1.55,-9.275,0.5042735042735043,0.45454545454545453,0.4661016949152542,0.3939393939393939
images/20191123164108.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.41856689453125,0.5204041280864198,20191123164108000,0.4171875,0.33611111111111114,0.05,0.08888888888888889,0.515625,0.65625,2.546875,-2.953125,0.75,-2.859375,,,0.5,0.0,1.640625,-2.640625,,,0.4830508474576271,0.4666666666666667
images/20191123164120.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.04120279947916667,0.16381655092592592,20191123164120000,,,,,,,,,,,,,,,,,,,0.5,0.36363636363636365
images/20191123164121.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.0030748155381944446,0.07585841049382716,20191123164121000,0.41875,0.3333333333333333,0.0484375,0.08611111111111111,0.532258064516129,0.6774193548387096,2.6451612903225805,-3.0161290322580645,0.8709677419354839,-2.9516129032258065,,,0.5645161290322581,-0.04838709677419355,1.8387096774193548,-2.725806451612903,,,0.5128205128205128,0.3888888888888889
images/20191123164109.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.3130696614583333,0.6255497685185185,20191123164109000,,,,,,,,,,,,,,,,,,,0.49572649572649574,0.4666666666666667
images/20191123164041.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4946072048611111,0.008538290895061728,20191123164041000,0.42109375,0.3458333333333333,0.04609375,0.08194444444444444,0.5084745762711864,0.6101694915254238,2.5084745762711864,-3.406779661016949,0.5423728813559322,-3.305084745762712,3.4745762711864407,-3.152542372881356,0.4745762711864407,-0.01694915254237288,1.4745762711864407,-3.0677966101694913,0.5,0.45454545454545453,0.4700854700854701,0.42424242424242425
images/20191123164055.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.8617404513888889,0.4562958140432099,20191123164055000,,,,,,,,,,,,,,,,,0.46846846846846846,0.45454545454545453,0.41025641025641024,0.5
images/20191123164057.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9966267903645833,0.4535228587962963,20191123164057000,0.2984375,0.4166666666666667,0.0359375,0.06388888888888888,0.5652173913043478,0.6739130434782609,6.826086956521739,-5.5,4.391304347826087,-5.391304347826087,7.934782608695652,-5.173913043478261,0.5434782608695652,0.0,5.391304347826087,-5.043478260869565,0.4473684210526316,0.5,0.39316239316239315,0.48484848484848486
images/20191123164122.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,,,,0.625,0.20277777777777778,0.03515625,0.0625,0.4888888888888889,0.5777777777777777,-2.2666666666666666,-2.066666666666667,-4.666666666666667,-1.9777777777777779,,,0.4222222222222222,0.0,-3.3555555555555556,-1.6666666666666667,,,0.5042735042735043,0.3888888888888889
images/20191123164056.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9674831814236111,0.4529393325617284,20191123164056000,0.47734375,0.8472222222222222,0.0578125,0.10277777777777777,0.4594594594594595,0.6081081081081081,1.135135135135135,-7.608108108108108,-0.33783783783783783,-7.527027027027027,1.8513513513513513,-7.418918918918919,0.3783783783783784,0.05405405405405406,0.2702702702702703,-7.324324324324325,0.4649122807017544,0.4666666666666667,0.39473684210526316,0.5
images/20191123164046.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5010498046875,0.9030092592592592,20191123164046000,0.4921875,0.9236111111111112,0.04140625,0.07361111111111111,0.5283018867924528,0.6415094339622641,1.0943396226415094,-11.452830188679245,-1.0377358490566038,-11.339622641509434,2.1320754716981134,-11.245283018867925,0.5849056603773585,-0.05660377358490566,-0.07547169811320754,-11.132075471698114,0.4954954954954955,0.4230769230769231,0.4473684210526316,0.4074074074074074
images/20191123164053.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5678941514756944,0.4632233796296296,20191123164053000,0.4234375,0.3472222222222222,0.04453125,0.07916666666666666,0.5087719298245614,0.631578947368421,2.6842105263157894,-3.508771929824561,0.6842105263157895,-3.43859649122807,3.6491228070175437,-3.2982456140350878,0.5263157894736842,0.07017543859649122,1.5789473684210527,-3.175438596491228,0.4824561403508772,0.4444444444444444,0.4358974358974359,0.5
images/20191123164053.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.6799370659722223,0.45665991512345677,20191123164053000,0.4234375,0.3472222222222222,0.04453125,0.07916666666666666,0.5087719298245614,0.631578947368421,2.6842105263157894,-3.508771929824561,0.6842105263157895,-3.43859649122807,3.6491228070175437,-3.2982456140350878,0.5263157894736842,0.07017543859649122,1.5789473684210527,-3.175438596491228,0.4824561403508772,0.4444444444444444,0.4358974358974359,0.5
images/20191123164045.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5021687825520833,0.7499300733024692,20191123164045000,0.41796875,0.3402777777777778,0.04921875,0.0875,0.5079365079365079,0.6666666666666666,2.4126984126984126,-3.015873015873016,0.6031746031746031,-2.9047619047619047,3.3333333333333335,-2.8412698412698414,0.5079365079365079,-0.015873015873015872,1.4603174603174602,-2.7301587301587302,0.5087719298245614,0.4074074074074074,0.46153846153846156,0.4074074074074074
images/20191123164119.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.12300075954861112,0.26436631944444444,20191123164119000,0.490625,0.9138888888888889,0.0421875,0.075,0.5185185185185185,0.6296296296296297,1.3703703703703705,-11.055555555555555,-0.6111111111111112,-11.12962962962963,,,0.5185185185185185,-0.07407407407407407,0.46296296296296297,-10.87037037037037,,,0.5087719298245614,0.3888888888888889
images/20191123164118.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,,,,,,,,,,,,,,,,,,,,,,0.48717948717948717,0.46153846153846156
images/20191123164044.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4956515842013889,0.4751880787037037,20191123164044000,0.36484375,0.5055555555555555,0.03984375,0.07083333333333333,0.5098039215686274,0.6470588235294118,4.294117647058823,-6.0588235294117645,2.0392156862745097,-5.9411764705882355,5.411764705882353,-5.823529411764706,0.49019607843137253,-0.0196078431372549,3.1176470588235294,-5.705882352941177,0.5,0.46153846153846156,0.4661016949152542,0.4444444444444444
images/20191123164044.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4989040798611111,0.6463903356481482,20191123164044000,0.36484375,0.5055555555555555,0.03984375,0.07083333333333333,0.5098039215686274,0.6470588235294118,4.294117647058823,-6.0588235294117645,2.0392156862745097,-5.9411764705882355,5.411764705882353,-5.823529411764706,0.49019607843137253,-0.0196078431372549,3.1176470588235294,-5.705882352941177,0.5,0.46153846153846156,0.4661016949152542,0.4444444444444444
images/20191123164049.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5023749457465277,0.975096450617284,20191123164049000,0.5640625,0.12638888888888888,0.0671875,0.11944444444444445,0.5,0.5930232558139535,-0.4069767441860465,-0.37209302325581395,-1.7093023255813953,-0.3023255813953488,0.2441860465116279,-0.2558139534883721,0.4418604651162791,0.0,-1.1162790697674418,-0.1744186046511628,0.5045045045045045,0.38461538461538464,0.4473684210526316,0.4074074074074074
images/20191123164101.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9917209201388889,0.08969425154320988,20191123164101000,,,,,,,,,,,,,,,,,0.45614035087719296,0.5,0.4017094017094017,0.5
images/20191123164115.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.15923258463541667,0.5026982060185186,20191123164115000,0.42109375,0.3388888888888889,0.0453125,0.08055555555555556,0.5172413793103449,0.6551724137931034,2.7758620689655173,-3.293103448275862,0.8275862068965517,-3.1724137931034484,,,0.5,0.0,1.8448275862068966,-2.9482758620689653,,,0.48760330578512395,0.3939393939393939
images/20191123164115.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.24997016059027777,0.49330391589506173,20191123164115000,0.42109375,0.3388888888888889,0.0453125,0.08055555555555556,0.5172413793103449,0.6551724137931034,2.7758620689655173,-3.293103448275862,0.8275862068965517,-3.1724137931034484,,,0.5,0.0,1.8448275862068966,-2.9482758620689653,,,0.48760330578512395,0.3939393939393939
images/20191123164114.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.012440321180555556,0.4955415702160494,20191123164114000,0.4234375,0.34305555555555556,0.04453125,0.07916666666666666,0.49122807017543857,0.6666666666666666,2.9298245614035086,-3.3859649122807016,1.0,-3.263157894736842,,,0.5263157894736842,0.017543859649122806,2.017543859649123,-3.0701754385964914,,,0.5087719298245614,0.4074074074074074
images/20191123164100.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9932318793402778,0.19716435185185185,20191123164100000,0.5765625,0.12777777777777777,0.0640625,0.11388888888888889,0.5,0.5853658536585366,-0.4878048780487805,-0.5853658536585366,-1.853658536585366,-0.524390243902439,0.14634146341463414,-0.4024390243902439,0.4268292682926829,0.012195121951219513,-1.2926829268292683,-0.3170731707317073,0.45614035087719296,0.45454545454545453,0.39316239316239315,0.5151515151515151
images/20191123164128.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.19443766276041666,0.2203342013888889,20191123164128000,0.62578125,0.1625,0.03828125,0.06805555555555555,0.4897959183673469,0.5918367346938775,-1.9183673469387754,-1.4081632653061225,-4.204081632653061,-1.2448979591836735,,,0.46938775510204084,0.0,-3.020408163265306,-0.8979591836734694,,,0.49572649572649574,0.4722222222222222
images/20191123164116.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.3717041015625,0.4934968171296296,20191123164116000,,,,,,,,,,,,,,,,,,,0.4661016949152542,0.4
images/20191123164102.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.8995917426215277,0.1859206211419753,20191123164102000,,,,,,,,,,,,,,,,,0.46153846153846156,0.5277777777777778,0.4152542372881356,0.5
images/20191123164103.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.8199896918402778,0.267163387345679,20191123164103000,0.07265625,0.2375,0.0984375,0.175,0.5079365079365079,0.6825396825396826,4.76984126984127,-1.0,3.873015873015873,-0.9523809523809523,5.2063492063492065,-0.873015873015873,0.5714285714285714,0.0,4.277777777777778,-0.8174603174603174,0.4700854700854701,0.48484848484848486,0.4322033898305085,0.5151515151515151
images/20191123164103.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.7503879123263889,0.3295187114197531,20191123164103000,0.07265625,0.2375,0.0984375,0.175,0.5079365079365079,0.6825396825396826,4.76984126984127,-1.0,3.873015873015873,-0.9523809523809523,5.2063492063492065,-0.873015873015873,0.5714285714285714,0.0,4.277777777777778,-0.8174603174603174,0.4700854700854701,0.48484848484848486,0.4322033898305085,0.5151515151515151
images/20191123164117.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.45227186414930554,0.49248890817901236,20191123164117000,,,,,,,,,,,,,,,,,,,0.4576271186440678,0.3939393939393939
images/20191123164106.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.4854342990451389,0.4798948688271605,20191123164106000,,,,,,,,,,,,,,,,,0.4954954954954955,0.43333333333333335,0.4661016949152542,0.45454545454545453
images/20191123164112.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.010828993055555555,0.9201244212962963,20191123164112000,0.4578125,0.0,0.190625,0.3388888888888889,0.5409836065573771,0.639344262295082,0.48770491803278687,0.24180327868852458,0.03278688524590164,0.2581967213114754,,,0.610655737704918,-0.02459016393442623,0.27459016393442626,0.3114754098360656,,,0.5042735042735043,0.43333333333333335
images/20191123164110.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.19084879557291667,0.7333309220679012,20191123164110000,0.465625,0.0,0.175,0.3111111111111111,0.5133928571428571,0.6696428571428571,0.4642857142857143,0.24107142857142858,-0.03125,0.26339285714285715,,,0.6294642857142857,-0.03571428571428571,0.22767857142857142,0.33035714285714285,,,0.4915254237288136,0.45454545454545453
images/20191123164111.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.06061604817708333,0.8413218557098765,20191123164111000,0.459375,0.0,0.18671875,0.33194444444444443,0.5397489539748954,0.6527196652719666,0.4769874476987448,0.2384937238493724,0.02092050209205021,0.25523012552301255,,,0.6150627615062761,-0.03347280334728033,0.26359832635983266,0.3138075313807531,,,0.49572649572649574,0.42424242424242425
images/20191123164105.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,,,,0.19375,0.7277777777777777,0.03359375,0.059722222222222225,0.5116279069767442,0.6511627906976745,10.116279069767442,-10.953488372093023,7.465116279069767,-10.837209302325581,11.418604651162791,-10.627906976744185,0.4418604651162791,-0.046511627906976744,8.720930232558139,-10.488372093023257,0.49122807017543857,0.4666666666666667,0.4576271186440678,0.45454545454545453
images/20191123164059.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.9953030056423611,0.314209587191358,20191123164059000,0.00234375,0.6083333333333333,0.03984375,0.07083333333333333,0.43137254901960786,0.6274509803921569,13.647058823529411,-7.705882352941177,11.431372549019608,-7.588235294117647,14.627450980392156,-7.411764705882353,0.5098039215686274,0.0196078431372549,12.313725490196079,-7.2745098039215685,0.45045045045045046,0.5,0.38461538461538464,0.5333333333333333
images/20191123164038.jpg,data/raw/capture1/positions.csv,data/landmarks/1/capture1/landmarks.csv,0.5004991319444444,0.1840253665123457,20191123164038000,0.42109375,0.3472222222222222,0.04453125,0.07916666666666666,0.5263157894736842,0.631578947368421,2.5789473684210527,-3.473684210526316,0.5263157894736842,-3.3508771929824563,3.5789473684210527,-3.2280701754385963,0.49122807017543857,-0.03508771929824561,1.5263157894736843,-3.1052631578947367,0.5,0.4666666666666667,0.4830508474576271,0.4666666666666667
